src/index.ts

```diff
@@ -18,14 +21,11 @@ const renderUsers = async (config: ApiConfig) => {
 };
 
 const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
```

Comment:
Why do you remove this code?
Since you have access to the ioc container here, you can register API config: `ioc.register('config', config);`

Reply:
Okey, thanks for your hint. I haven't seen code like this before, so I thought maybe it's unnecessary